custom_components/frigate/const.py

```diff
@@ -2,7 +2,7 @@
 # Base component constants
 NAME = "Frigate"
 DOMAIN = "frigate"
-FRIGATE_VERSION_ERROR_CUTOFF = "0.8.4"
+FRIGATE_VERSION_ERROR_CUTOFF = "0.13.0"
```

Comment:
@NickM-27 Do you know what the next release version will be? If we could be more precise than this it would avoid log spam for early users.

Reply:
I don't see any reason the full release wouldn't be 0.13.0

Reply:
What I meant was if we use this cutoff, but you use another beta tag (e.g. `v0.13.0-beta3`) I think it will spam the logs erroneously, since I suspect `v0.13.0-beta3` is `<` `0.13.0` . If we set this to `v0.13.0-beta3` we'd avoid that problem in this scenario.

Reply:
What it does now is that the version check passes for all beta versions of 0.13.0, so it will not spam the logs for the beta testers. Only caveat is that users with 0.13.0 beta1 and beta2 will not be notified to upgrade their Frigate instance. Not sure if that is acceptable?

Reply:
We should probably just leave the logic was it was before (with `<=` and make it `0.12.1` that way any users running integration 5.0 and 0.12.1 or later will get logs

Reply:
But users with integration 5.0 and Frigate 0.12.1 have a [broken](https://github.com/blakeblackshear/frigate-hass-integration/pull/555#discussion_r1362210538) integration...

Reply:
It used to be

```
    if AwesomeVersion(server_version.split("-")[0]) <= AwesomeVersion(
        0.8.4
    ):
```

You changed it to

```
    if AwesomeVersion(server_version.split("-")[0]) < AwesomeVersion(
        0.13.0
    ):
```

and I am suggesting it should remain consistent with what it was before

```
    if AwesomeVersion(server_version.split("-")[0]) <= AwesomeVersion(
        0.12.1
    ):
```

Reply:
You're right.... But they will be advised to install version 0.12.1, which they already have running.

Reply:
No they won't, 

```
        _LOGGER.error(
            "Using a Frigate server (%s) with version %s <= %s which is not "
            "compatible -- you must upgrade: %s",
            entry.data[CONF_URL],
            server_version,
            FRIGATE_VERSION_ERROR_CUTOFF,
            FRIGATE_RELEASES_URL,
        )
```

it will simply say that they are `running a frigate server with version 0.12.1 <= 0.12.1 which is not compatible, you must upgrade` and that would be expected for a user running the beta integration

Reply:
OK. I thought it stated "you must upgrade **to**: %s". I will revert the version check logic as suggested.